packages/lesswrong/lib/collections/comments/callbacks.js

```diff
@@ -378,4 +402,4 @@ async function updateTopLevelCommentLastCommentedAt (comment) {
   Comments.update({ _id: comment.topLevelCommentId }, { $set: {lastSubthreadActivity: new Date()}})
```

Comment:
Two weird things about this. First: this only sets `lastSubthreadActivity` on root comments, but nothing in the name or any comments suggests that the field isn't valid everywhere. And second, if `topLevelCommentId` is missing, this will try to update `id:null` which matches the entire databse. (It won't bring down the server because this doesn't have `multi:true`, but it's pretty sketchy).